### DISCUSSION TITLE
Add "test" severity level

### DISCUSSION THREAD

**Discussion by LTLanda** [Security-Onion-Solutions/securityonion#6509] [Unsupported Versions] [answered]
Hi SO,

I've noticed that the severity-levels in playbook.py in Soctopus service are hard coded.
Is there an option to add a "test" severity level? I want to create test alerts and not mix them up with the prod alerts. In other words, I would like to be able to filter (or filter out) all the test cases in the Alerts panel.

**Comment by defensivedepth** [comment, marked as answer]
These severity levels are tied to the Sigma spec definition of `level` - https://github.com/SigmaHQ/sigma/wiki/Specification#level

We will look into whether or not this is something that could be user-definable rather than hardcoded.
